python/django/python-guestbook/kubernetes-manifests/guestbook-postgres.deployment.yaml

```diff
@@ -28,3 +28,6 @@ spec:
           name: db
           ports:
             - containerPort: 5432
+          env:
+            - name: POSTGRES_HOST_AUTH_METHOD
+              value: "trust"
```

Comment:
Don't we need the same to be set on hello-world sample for Python (django version)?

Reply:
We don't since the hello world Django example doesn't have a Postgres deployment. Thanks for checking.